paper-card.html

```diff
@@ -102,44 +102,47 @@
         font-weight: 400;
         color: var(--paper-card-header-color, #000);
 
-        @apply(--paper-card-header-text);
+        @apply --paper-card-header-text;
       }
 
       .header .title-text.over-image {
         position: absolute;
         bottom: 0px;
 
-        @apply(--paper-card-header-image-text);
+        @apply --paper-card-header-image-text;
       }
 
-      :host ::content .card-content {
+      :host .card-content ::slotted(*) {
         padding: 16px;
         position:relative;
 
-        @apply(--paper-card-content);
+        @apply --paper-card-content;
       }
 
-      :host ::content .card-actions {
+      :host .card-actions ::slotted(*) {
         border-top: 1px solid #e8e8e8;
         padding: 5px 16px;
         position:relative;
 
-        @apply(--paper-card-actions);
+        @apply --paper-card-actions;
       }
     </style>
 
-    <div class="header">
-      <iron-image
-          hidden$="[[!image]]"
-          src="[[image]]"
-          alt="[[alt]]"
-          placeholder="[[placeholderImage]]"
-          preload="[[preloadImage]]"
-          fade="[[fadeImage]]"></iron-image>
-      <div hidden$="[[!heading]]" class$="title-text [[_computeHeadingClass(image)]]">[[heading]]</div>
-    </div>
-
-    <content></content>
+    <div class="paper-material" elevation="1">
```

Comment:
Why are the shadows not directly set on `:host` anymore? In one of our templates (Pesto), I had a style rule that assumed the shadow would be on `<paper-card>`:

```css
paper-card {
  padding: 16px 0 16px 100px;
}
```